sbpy/photometry/tests/test_core.py

```diff
@@ -0,0 +1,233 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+import numpy as np
+from astropy import units as u
+from astropy.modeling import Parameter
+from ..core import *
+
+def test_ref2mag():
+    assert np.isclose(ref2mag(0.1/np.pi, 460), 3.3208379018205285)
+    assert u.isclose(ref2mag(0.1/np.pi, 460*u.km), 3.3208379018205285*u.mag)
```

Comment:
`u.isclose` has been implemented in astropy version 3.0.2, causing a build fail for older versions. I suggest that you test the version of astropy and ignore the `u.isclose` test if the version is lower than 3.0.2. You can use the following code for this 

```
from distutils.version import LooseVersion
if (LooseVersion(astropy.__version__) >= LooseVersion('3.0.2')):
   ...
```